templates/partials/character-header.html

```diff
@@ -140,119 +150,136 @@
+            <a class="button-roll death-roll">{{localize "WITCHER.Actor.SavingThrow"}}</a>
+            <a class="button-roll heal-button">{{localize "WITCHER.Heal.button"}}</a>
         </div>
-        <div class="death-section">
-            <a class="death-minus"><i class="fas fa-minus"></i></a>
-            {{#if (gt system.deathSaves 0)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 0)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 1)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 1)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 2)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 2)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 3)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 3)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 4)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 4)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 5)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 5)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 6)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 6)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 7)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 7)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 8)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 8)}}<i class="fas fa-skull"></i>.{{/unless}}
-            <a class="death-plus"><i class="fas fa-plus"></i></a>
+        <div class="char-button-collum">
+            <a class="button-roll crit-roll">{{localize "WITCHER.Actor.Crit/Fumble"}}</a>
+            <a class="button-roll defence-roll">{{localize "WITCHER.Actor.Defense"}}</a>
+            {{#if useVerbalCombat}}
+                <a class="button-roll verbal-button">{{localize "WITCHER.verbalCombat.Title"}}</a>
+            {{/if}}
         </div>
-        <div class="right-bottom">
-            <div class="right-row">
-                <div class="right-element">
-                    <label>{{localize "WITCHER.Actor.Hp"}} ({{system.derivedStats.hp.max}})/{{system.derivedStats.hp.unmodifiedMax}}</label>
-                    {{#if system.deathStateApplied}}
+    </div>
+    <div class="death-section">
+        <a class="death-minus"><i class="fas fa-minus"></i></a>
+        {{#if (gt system.deathSaves 0)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless
+            (gt system.deathSaves 0)}}<i class="fas fa-skull"></i>.{{/unless}}
```

Comment:
if it's possible we could use if else handle bars here

```suggestion
        {{#if (gt system.deathSaves 0)}}<i class="death fas fa-skull-crossbones"></i>.{{else}}<i class="fas fa-skull"></i>.{{/if}}
```

Reply:
Thank you for noticing. Will do